i18n/en.json

```diff
@@ -167,6 +174,8 @@
 	"wantedproperties": "Wanted properties",
 	"smw-wantedproperties-docu": "This page lists [https://www.semantic-mediawiki.org/wiki/Wanted_properties wanted properties] that are used in the wiki but do not have a page describing them. For a differentiated view, see the [[Special:Properties|entire]] or  [[Special:UnusedProperties|unused properties]] special pages.",
```

Comment:
@mwjames Now I am pretty sure that we are talking about duplicated system messages. Shoudn'd the old strings be removed. Now we have the situation that the translators have to translate the messages again instead of the twn people just moving the old message key to the new one.